routes/user.js

```diff
@@ -360,7 +360,8 @@ router.get('/playlist/:playlist_name/export3', function(req, res) {
                     if (i === arr.length - 1) {
                         console.log(songss)
                         res.render('export3', {
-                            songss: songss
+                            songss: songss,
```

Comment:
i think it would be better to rename these variables from songss to something more descriptive.